specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizerReceiptOcr.json

```diff
@@ -1,222 +0,0 @@
-{
```

Comment:
the file is removed intentionally? although it's a preview version, to double confirm

Reply:
Yes.  We previously used external references.  We now merged the content into FormRecognizerReceipt.json so there are no external dependencies.